modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -48,10 +51,55 @@ initrd --name main http://<HTTP_server>/rhcos-<version>-live-initramfs.<architec
 <1> Specify locations of the {op-system} files that you uploaded to your HTTP server. The `kernel` parameter value is the location of the `kernel` file, the `initrd=main` argument is needed for booting on UEFI systems, the `coreos.inst.ignition_url` parameter value is the location of the worker Ignition config file, and the `coreos.live.rootfs_url` parameter value is the location of the live `rootfs` file. The `coreos.inst.ignition_url` and `coreos.live.rootfs_url` parameters only support HTTP and HTTPS.
 <2> Specify the location of the `initramfs` file that you uploaded to your HTTP server.
 +
-+
 [NOTE]
 ====
 This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
 ====
 
+** For iPXE (`x86_64` + `aarch64`):
```

Comment:
Is the second IPXE commands on line 45 of this file necessary for the multi-arch procedure?